CHANGES.rst

```diff
@@ -4,7 +4,11 @@ Change History
 4.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Python 3.7 and 3.8-dev.
+
+- Drop support for Python 3.4.
+
+- Fix PyPy version for TravisCI.
```

Comment:
Travis CI has both `pypy` and `pypy3` available for xenial.

Reply:
Now, both gets tested.